docs/source/cloudlabtutorial/index.rst

```diff
@@ -0,0 +1,126 @@
+Cloudlab Tutorial 
+=====================================
+
+Getting Started
+-----------------
+
+- This tutorial assumes that you have access to the openNetVM `CloudLab <https://cloudlab.us/>`_ 
+    - To gain access, you must first create an account, choose to join a project and enter "GWCloudLab" to request access
```

Comment:
We only want GW students to use the "GWCloudLab" organization, everyone else will need one from their own university.
```suggestion
    - To gain access, you must first create an account, and join a project from your university. See `the CloudLab Docs http://docs.cloudlab.us/users.html` for more information
```